### ISSUE TITLE
source checksum

### ISSUE THREAD

**Issue by theamericanaccount** [sphinx-contrib/sphinxcontrib-towncrier#95]
hello,
ive noticed a discrepancy between the source checksum on here and in the archlinux pkgbuild.
from both mine and arch tree it results the checksum should be another and it should have not changed since 2023.
my sources are published at
https://github.com/themartiancompany/python-sphinxcontrib-towncrier-ur
both my builds and arch one are from way later source publishing and no change seems have been going on github since then so I was wondering how and why the checksum changed.
im puzzled how the package actually built in the first place.
has there been a forced repo update?

**Comment by webknjaz** [comment]
Assuming you're talking about Git archive, this might be related to `setuptools-scm`. Otherwise, there's not enough context to make meaningful educated guesses.
The distribution can be built using any standard PEP 517 build frontend.

**Comment by theamericanaccount** [comment]
I'm talking about the released source tarball here on Github.
I have changed my sources to use the Pypi one because I remembered it doesn't allow for re-releases and it also reports the checksum as I didn't want to check all the source.
The checksum of the tarball must have changed months after their publication anyway.
I have checked your CI logs and they also report the file should have not changed
I had never stumbled onto such a weird situation myself.
